pkg/acquisition/modules/syslog/syslog.go

```diff
@@ -199,38 +183,24 @@ func (s *SyslogSource) handleSyslogMsg(out chan types.Event, t *tomb.Tomb, c cha
 			logger := s.logger.WithField("client", syslogLine.Client)
 			logger.Tracef("raw: %s", syslogLine)
 			linesReceived.With(prometheus.Labels{"source": syslogLine.Client}).Inc()
-			p := rfc5424.NewParser()
-			m, err := p.Parse(syslogLine.Message)
+			p := rfc3164.NewRFC3164Parser(rfc3164.WithCurrentYear())
+			err := p.Parse(syslogLine.Message)
 			if err != nil {
-				logger.Debugf("could not parse as RFC5424 (%s)", err)
-				p = rfc3164.NewParser(rfc3164.WithYear(rfc3164.CurrentYear{}))
-				m, err = p.Parse(syslogLine.Message)
+				logger.Debugf("could not parse as RFC3164 (%s)", err)
+				p2 := rfc5424.NewRFC5424Parser()
+				err = p2.Parse(syslogLine.Message)
 				if err != nil {
 					logger.Errorf("could not parse message: %s", err)
 					logger.Debugf("could not parse as RFC3164 (%s) : %s", err, syslogLine.Message)
```

Comment:
might not be related but given that we're pushing our parser, we want to allow users to log unparsed messages by setting the log_source to debug level so that they can open us issues :)

Reply:
Raw messages are logged at the trace level because it can get very verbose very quickly (too much for debug level IMO)